nus-faster-pay/src/Screens/AccountDashboard/Components/PaymentCard.tsx

```diff
@@ -4,6 +4,8 @@ import { Body } from "../../../Utils/SharedComponents";
 import { COLORS } from "../../../Utils/Colors";
 import { RightArrow } from "./RightArrow";
 import { useLocation, useNavigate } from "react-router-dom";
+import { useContext } from "react";
```

Comment:
lgtm!! just wanted to ask, is there a reason you chose to use context instead of passing around the cart items together with ivConfig? I was thinking to keep things cleaner, if context works well, i might bundle the ivConfig in a context instead too